libs/output-mapping/src/Writer/TableWriter.php

```diff
@@ -239,21 +262,30 @@ private function createLoadTableTask(
         return $loadTask;
     }
 
-    private function ensureValidDestinationBucketExists(MappingDestination $destination, array $systemMetadata)
+    private function ensureValidDestinationBucketExists(MappingDestination $destination, array $systemMetadata): array
     {
         $destinationBucketId = $destination->getBucketId();
-        $destinationBucketExists = $this->clientWrapper->getBasicClient()->bucketExists($destinationBucketId);
+        try {
+            $destinationBucketDetails = $this->clientWrapper->getBasicClient()->getBucket($destinationBucketId);
+            $destinationBucketExists = true;
+        } catch (ClientException $e) {
+            if ($e->getCode() == 404) {
+                $destinationBucketExists = false;
+            }
+            throw $e;
```

Comment:
why?

Reply:
why is it thrown?  it was being thrown before in the `bucketExists` method

Reply:
no it wasn't - that doesn't make sense, why would it set the $destinationBucketExists variable then?

Reply:
ah fuck, right!  it returns there 🤦 .  thanks odin